src/main/java/com/authzed/api/v1/RetryableClient.java

```diff
@@ -0,0 +1,499 @@
+     * 
+     * The conflict strategy can be one of:
+     * - FAIL - will return an error if any duplicate relationships are found.
+     * - SKIP - will ignore duplicates and continue with the import.
+     * - TOUCH - will retry the import with TOUCH semantics in case of duplicates.
+     * 
+     * @param relationships The relationships to import
+     * @param conflictStrategy The conflict strategy to use
+     * @param timeoutSeconds The timeout for the operation in seconds
+     * @return The number of relationships loaded
+     * @throws Exception If there are errors during import
+     */
+    public long retryableBulkImportRelationships(
+            List<Relationship> relationships,
+            ConflictStrategy conflictStrategy,
+            int timeoutSeconds) throws Exception {
+        
+        final BulkImportResponseCollector responseCollector = new BulkImportResponseCollector();
+        final StreamObserver<BulkImportRelationshipsResponse> responseObserver = new StreamObserver<BulkImportRelationshipsResponse>() {
+            @Override
+            public void onNext(BulkImportRelationshipsResponse response) {
+                responseCollector.addResponse(response);
+            }
+            
+            @Override
+            public void onError(Throwable t) {
+                responseCollector.setError(t);
+            }
+            
+            @Override
+            public void onCompleted() {
+                responseCollector.setCompleted();
+            }
+        };
+        
+        // Create initial request
+        BulkImportRelationshipsRequest request = BulkImportRelationshipsRequest.newBuilder()
+                .addAllRelationships(relationships)
+                .build();
+        
+        // Send the request
+        StreamObserver<BulkImportRelationshipsRequest> requestObserver = 
+                asyncExperimentalService.bulkImportRelationships(responseObserver);
+        
+        try {
+            requestObserver.onNext(request);
+            requestObserver.onCompleted();
+            
+            // Wait for response to complete
+            BulkImportRelationshipsResponse response = responseCollector.await();
+            return response.getNumLoaded();
+        } catch (Throwable throwable) {
+            logger.log(Level.INFO, "Handling error in retryableBulkImportRelationships: " + throwable.getMessage());
+            
+            // Handle the error based on its type and the chosen conflict strategy
+            if (isCanceledError(throwable)) {
+                throw new RuntimeException("Request canceled", throwable);
+            }
+            
+            if (isAlreadyExistsError(throwable)) {
+                if (conflictStrategy == ConflictStrategy.SKIP) {
+                    // Skip conflicts - return success
+                    logger.log(Level.INFO, "ALREADY_EXISTS detected with SKIP strategy - returning success");
+                    return relationships.size();
```

Comment:
What does this actually return? Is this function supposed to return void or is it implicitly typed?